src/key.js

```diff
@@ -11,7 +11,7 @@ var h = syn.helpers,
 		var result;
 
 		try {
-			result = el.selectionStart !== undefined;
+			result = ((el.selectionStart !== undefined) && (el.selectionStart !== null));
```

Comment:
Hi Chasen, its a good catch. 

My confidence in my js is lower than other languages so I might be adding unnecessary formatting in places. Thanks for calling it out.